.github/workflows/linux.yml

```diff
@@ -35,14 +35,15 @@ jobs:
     container:
       image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
       volumes: [ "/mount:/mount" ]
+      options: -v ${{ github.workspace }}:${{ github.workspace }}
```

Comment:
Isn't it equivalent to passing a new key-value pair to "volumes: [ "/mount:/mount" ]" in the previous line?